container/pull.bzl

```diff
@@ -71,7 +71,11 @@ container_import(
             tag=repository_ctx.attr.tag)
     ]
 
-  result = repository_ctx.execute(args)
+  timeout = None
+  if "TIME_OUT" in repository_ctx.os.environ:
+      timeout = int(repository_ctx.os.environ.get("TIME_OUT"))
+
+  result = repository_ctx.execute(args, timeout = timeout)
```

Comment:
I think that you could fix this to respect the default value `execute()` provides via:
```python
kwargs = {}
if ...
  kwargs["timeout"] = int(...)
result = rctx.execute(args, **kwargs)
```